flytekit/extras/tasks/shell.py

```diff
@@ -237,3 +246,28 @@ def execute(self, **kwargs) -> typing.Any:
 
     def post_execute(self, user_params: ExecutionParameters, rval: typing.Any) -> typing.Any:
         return self._config_task_instance.post_execute(user_params, rval)
+
+
+portable_shell_task = ShellTask(
```

Comment:
rather than messing with the class, we can just create a shell task instance which is wrapper around our desired "pure" shell script. Since most shell scripts being ported over will need the same features (environment, arguments), we can support that within the template. The template can be expanded, and new templates can be made to handle other use-cases.